src/sentry/lang/javascript/processor.py

```diff
@@ -66,6 +66,16 @@ class ZeroReturnError(Exception):
 logger = logging.getLogger(__name__)
 
 
+def expose_url(url):
+    if url is None:
+        return u'<unknown>'
+    if url.startswith('data:'):
+        return u'<data url>'
```

Comment:
maybe "data:<redacted>" would be better?

minor, and i dont care that much (esp since its merged already)